vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -19495,6 +19565,15 @@ unit:V-SEC-PER-M
   rdfs:label "V SEC PER M" ;
   skos:prefLabel "volt second per metre" ;
 .
+unit:V2-HR
+  a qudt:Unit ;
+  qudt:conversionMultiplier 3.6e03 ;
+  qudt:conversionOffset "0.0"^^xsd:double ;
+  qudt:expression "v^2hr" ;
+  qudt:hasQuantityKind quantitykind:LorenzCoefficient ;
```

Comment:
How can this unit have the same quantity kind as unit:V2-PER-K2? Something is wrong.

Reply:
I must have missed this. I took the DV for Volt: A0E-1L2I0M1H0T-3D0. If I square this then I would double the exponents wouldn't I? That would make: A0E-2L4I0M2H0T-6D0. If I then multiply by T1 I should get: A0E-2L4I0M2H0T-5D0. There is no such dimension vector in QUDT (for V2 'or' V2-HR).

Here is a reference:

https://patents.google.com/patent/US4066959A/en

In the metering of electrical energy it is often advantageous to know the number of volt-square-hours (V2 hr) as, for example, in the measurement of apparent electrical power and energy. Apparent electrical power is the product of current and voltage, oftentimes expressed in kilovolt-amperes (kVa), and is independent of the phase angle between the current and voltage. Apparent electrical energy is expressed in kilovolt-ampere-hours (kVah) which is also independent of the phase angle between the current and the voltage. As is known in the art, both apparent electrical energy and apparent electrical power are proportional to V2 hr and V2, respectively. In many instances, therefore, a simplified way of determining apparent electrical power or energy in a system is to determine the number of V2 hrs.